modules/ztp-precaching-downloading-artifacts.adoc

```diff
@@ -198,22 +186,21 @@ You need to include the {rh-rhacm} hub version, so that the {factory-prestaging-
 +
 [source,terminal]
 ----
-# podman run -v /mnt:/mnt -v /root/.docker:/root/.docker --privileged --rm quay.io/openshift-kni/telco-ran-tools:latest -- factory-precaching-cli download \
--r 4.11.5 \ <1>
---hub-version 2.5.4 \ <2>
--f /mnt \ <3>
---ai-img registry.redhat.io/multicluster-engine/assisted-installer-agent-rhel8@sha256:da1753f9fcb9e229d0a68de03fac90d15023e647a8db531ae489eb93845d5306 \ <4>
---ai-img registry.redhat.io/multicluster-engine/assisted-installer-reporter-rhel8@sha256:e8d6b78248352b1a8e05a22308185a468d4a139682d997a7f968b329abbc02cd \ <4>
---ai-img registry.redhat.io/multicluster-engine/assisted-installer-rhel8@sha256:33abd6e21cfdc36dd4337fa6f3c3442d33fc3f976471614dca5b8ef749e7a027 \ <4>
---img quay.io/custom/repository \ <5>
---du-profile -s <6>
-----
-<1> Defines the {product-title} release version.
-<2> Defines the version of the hub cluster.
-<3> Defines the folder where you want to pre-cache the images on the disk.
-<4> Defines the MCE and AI artifacts required for provisioning.
-<5> Defines the repository where you store your additional images. These images are downloaded and pre-cached on the disk.
-<6> The `--du-profile` argument specifies pre-caching the Operators included in the DU configuration.
+# podman run -v /mnt:/mnt -v /root/.docker:/root/.docker --privileged --rm quay.io/openshift-kni/telco-ran-tools:latest --    factory-precaching-cli download \ <1>
```

Comment:
This has some extra space in it between "--" and "factory-precaching-cli". Probably not critical, but since the line is so long it would probably be good to remove.